src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/production/turbines/GT_MTE_LargeTurbine_Plasma.java

```diff
@@ -163,7 +164,7 @@ protected boolean filtersFluid() {
                     optFlow = MathUtils.roundToClosestInt(aTotalOptimalFlow);
                     euPerTurbine = MathUtils.roundToClosestInt(aEUPerTurbine);
                     if (optFlow <= 0 || baseEff <= 0) {
-                        stopMachine(); // in case the turbine got removed
+                        stopMachine(ShutDownReasonRegistry.NONE); // in case the turbine got removed
```

Comment:
`NO_TURBINE` suit better, doesn't it?

Reply:
This is unlikely to be called since turbine check has been handled in the above few lines. I think the shutdown here is just for safety